.circleci/config.yml

```diff
@@ -134,10 +134,15 @@ publish-steps: &publish-steps
           TAGSUFFIX=$(date +%Y%m%d)
           docker login -u ${DOCKER_USER} -p ${DOCKER_PASSWORD}
           for image in "vl3_ucnf-nse" "ucnf-kiknos-vppagent"; do
-              docker tag ${ORG}/${image}:${CIRCLE_SHA1} ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}
-              docker push ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}
-              docker tag ${ORG}/${image}:${CIRCLE_SHA1} ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}-${TAGSUFFIX}
-              docker push ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}-${TAGSUFFIX}
+              if [[ ${CIRCLE_PROJECT_USERNAME} == "cisco-app-networking" ]]; then
+                docker tag ${ORG}/${image}:${CIRCLE_SHA1} ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}
+                docker push ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}
+                docker tag ${ORG}/${image}:${CIRCLE_SHA1} ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}-${TAGSUFFIX}
+                docker push ${DOCKER_USER}/${image}:${CIRCLE_BRANCH}-${TAGSUFFIX}
+              else
+                docker tag ${ORG}/${image}:${CIRCLE_SHA1} ${DOCKER_USER}/${image}:${CIRCLE_SHA1}
+                docker push ${DOCKER_USER}/${image}:${CIRCLE_SHA1}
+              fi
```

Comment:
IMO, this is extra complexity that's not required.  The publish job is only running against branches `master` and any `release*` which wouldn't be branches used in private forks unless someone is testing the publish job itself.  In that case, the user's circleci context could have their own DOCKER_USER/PASSWORD set for pushing to their private dockerhub repo.

https://github.com/cisco-app-networking/nsm-nse/pull/6/files#diff-1d37e48f9ceff6d8030570cd36286a61R214-R218